### DISCUSSION TITLE
LoadingPhase

### DISCUSSION THREAD

**Discussion by falbellaihi1** [Sixze/ALS-Refactored#114] [General]
Hey I have been getting into the issue of some nodes disappearing after each restart that I solved by changing the LoadingPhase and had to put it EarliestPossible so that I wouldn't lose the "BlendPosesByGameplayTag" I didnt think it would be much of contribution if I open a pull request for this only. 

Reason why I went into this issue is I had my animation blueprints inside my game content folder and not on the plugins folder so my reasonings are that the AnimBPs would load and look for "BlendPosesByGameTag" and wouldnt find it so it would then delete its references but luckily I just changed the LoadingPhase and all the references were on place

Thanks

**Comment by Sixze** [comment]
Are you sure this is the cause of your issue? I verified this by using a duplicate of `ABP_Als_Monolithic` placed in the project's Content folder, using `"LoadingPhase": "PostDefault"` to make the plugin load even later than everything else and packaging the project. In all cases, everything worked properly.

**Reply by falbellaihi1** [reply]
What I actually did was I moved the whole contend folder into my project (along with all the layers inside ALS->ALS->Character->AnimInstances) the issue was happened with the animations inside the Overlay folder, it confused me at first thought maybe its some redirect issue but then what I have done is 
put the "BlendPosesByGameTag" inside "ABP_Als_Injured" then I restarted the engine and it was gone, then I went into the .uplugin and changed the loadingPhase and done the same steps again and restarted the engine and all of missing nodes were placed again

**Reply by Sixze** [reply]
So, I created a new project, added ALS, moved all the plugin content to the project content folder, changed some animation blueprints and restarted the editor several times, and still everything looks fine.

**Reply by falbellaihi1** [reply]
I am not certainly sure but it seems to me like this is an issue from my side, I rechecked it with my project and as you can see from the below images lost all the connections to the nodes, however I did check with a new fresh project and moved the animations to the content folder did not get the issue :S 

But I will keep digging to find what exactly caused this to happen and keep you posted 
![image](https://user-images.githubusercontent.com/9819280/179047117-c0b97c42-2dbf-4fc8-8487-12d7e8a00213.png)


![UnrealEditor_CbHkeA7AWN](https://user-images.githubusercontent.com/9819280/179047247-ad0b57e4-5b54-4b61-8d34-6f282aa42ab1.png)

**Comment by Sixze** [comment]
I changed the plugin loading phase in 04a03d4137ae965d86cf90baf807a86d59eaae91, but I'm curious what the real cause of your issue is.

**Reply by falbellaihi1** [reply]
It's driving me nuts, I took my config folder and moved it to another project containing the same plugin, then I did the same procedure, added the same plugins as the ones in my main project. No go.

Then, I created a UE4 project and converted it to UE5 then added the plugin. No go.

Still, not able to reproduce but I am stubborn and I will find it, I just have to take a vacation now!😅
